translations/en-us.yml

```diff
@@ -2161,7 +2162,6 @@ routes:
         email: Email
 osf-components:
     addon-card:
```

Comment:
I think `disable` and the `disconnect-modal-xyz` translations are no longer needed as well.